### PR TITLE
Feature/2 cleanup

### PR DESCRIPTION
**Pull request by TE4-Daniel-Kull**
#2

### CODE REVIEW

**Comment by te4-hugo-svanberg** [comment]
Code smells bro. Fix that shit. 
:)

File: index.js [hunk 1]
```diff
@@ -1 +1,14 @@
-console.log("thank you for being an asshole...! :/")
\ No newline at end of file
+alert("Welcome traveler!")
+alert("This is the most useless site you will ever encounter...")
+alert("Better make the most of it! :D")
```
Comment (te4-hugo-svanberg) [inline index.js]: shut up

Comment (TE4-Daniel-Kull) [reply index.js]: sawy

Comment (TE4-Daniel-Kull) [reply index.js]: Fixed it

**Pull request merged**